src/xtopcom/xbasic/xsimple_message.hpp

```diff
@@ -19,7 +21,7 @@ NS_BEG1(top)
  * @tparam MessageT The message type.
  */
 template <typename MessageT>
-class xtop_simple_message final : public xhashable_t<xtop_simple_message<MessageT>> {
+class xtop_simple_message final : public xhashable_t<xtop_simple_message<MessageT>>, public xstatistic::xstatistic_obj_face_t {
```

Comment:
should the inheritance be ENABLED by defining CACHE_SIZE_STATISTIC or CACHE_SIZE_STATISTIC_MORE_DETAIL?

Reply:
no, threre is an empty base class for not defining CACHE_SIZE_STATISTIC or CACHE_SIZE_STATISTIC_MORE_DETAIL.